### ISSUE TITLE
Arbeitsgruppen/Tätigkeitsbereich eigene geschlossene Kalender Termine ermöglichen

### ISSUE THREAD

**Issue by Erdmut** [roman-agri/waedichoerbli-junto#15]

**Comment by padili** [comment]
Hier sehe ich eine Google Kalender Integration für die jeweiligen Bereiche, die das benötigen.
Ich denke, dies ist gewünscht für Member von:
Auslieferung, Tunnelpflege, Betriebsrat .. welche noch?

**Comment by padili** [comment]
@Erdmut jetzt habe ich diese erst gesehen auf der Einstiegsseite
![image](https://user-images.githubusercontent.com/25586157/149805593-d59b3adf-c820-437d-86e9-f813544248b3.png)
sind diese Kalender gemeint?

**Comment by Erdmut** [comment]
ja, diese Kalender sind gemeint. Die lassen sich so zwar als eigene Kalender anzeigen, aber sie erscheinen dann auch in der Gesamtübersicht und sind von allen nutzbar, was nicht sein darf für unsere Bedürfnisse, sonst gibts chaos, wenn sich da jeder in den Auslieferkalender zb. eintragen kann.  Weil es gibt immer zuviel Leute, die "machen einfach mal", ohne viel zu denken oder auch weil sie nicht alles wissen können. Derzeit haben wir individuelle Lösungen. Meist über Google, wie die Ausliefergruppe. Ziel: das die Termine auch hier im Junto buchbar sind und automatisch dann die Stunden angerechnet werden, wie es auch sonst im Junto läuft, aber eben als seperat zugänglicher Kalender.

Die Gruppen sind aktuell: Auslieferung, (evtl. Rezepte), Anzucht, Tunnel lüften und Wässern, die neue Pilzgruppe, In Zukunft evtl auch eine Kräutergruppe, Verarbeitungsgruppe, Kochgruppe, Beerengruppe, Ag für Veranstaltungen/Kulturelles und weitere.

**Comment by pascalfree** [comment]
Verstehe ich das richtig, dass es darum geht bestimmte Tätigkeitsbereiche und Einsätze nur für gewisse Mitglieder buchbar (und sichtbar) zu machen?

**Comment by Erdmut** [comment]
Genauer gesagt die Tätigkeitsbereichs-Einsätze sollen nur für Bereichsmitglieder sichtbar/buchbar sein. Dass der Bereich existiert und man sich da anmelden kann, sollte schon für alle sichtbar sein, aber ein Beitritt nur möglich auf Anfrage, also mit Bestätigung eines Admins. --> siehe  Issue 16   https://github.com/roman-agri/waedichoerbli-junto/issues/16

**Comment by padili** [comment]
Ich würde dazu bei der Einstiegsseite die allg. sichtbaren Tätigkeitsbereiche anzeigen und zusätzlich jene Bereiche wo der Benutzer eingetragen ist. Das sind dann solche die in der DB als nicht (allgemein) sichtbar markiert sind.

**Comment by padili** [comment]
ich versuche dies mal in einem separaten Menüpunkt zu implementieren, Dauert aber ein wenig, bis ich wieder drin bin 🙈 
Nachher bei 1.5 kann dies dann in den jeweiligen Seiten (bsp home) eingebunden werden 😃

**Comment by padili** [comment]
@Erdmut 
erste Version unter 
https://junto.waedichoerbli.ch/my/specialwork
ist sehr Waedichoerbli spezifisch, da es einfach durch die hidden areas durch geht und einen Link zeigt, wenn der User Member ist. Ganz sicher nicht für die Allgemeinheit verwendbar.

**Comment by Erdmut** [comment]
Hallo Patricia, ich hab es mir jetzt näher angesehen.
Das löst das Thema ganz gut, das die Leute sich nicht einfach in alle Tätigkeitsbereiche anmelden können, aber sehen, das sie existieren.
Jedoch ist ein Kalendereintrag einer solchen hidden Area, trotzdem für alle in der Terminübersicht drin bzw. anmeldbar, was beim Beispiel der Ausliefergruppe problematisch ist.
